policy/release/tasks.rego

```diff
@@ -197,11 +197,40 @@ current_required_tasks := task_data if {
 
 ## get the required task data for a pipeline with a label
 required_pipeline_task_data := task_data if {
-	some att in lib.pipelinerun_attestations
+	some att in _pipelinerun
 	count(tkn.tasks(att)) > 0
 	task_data := tkn.required_task_list(att)
 }
 
-_status(task) := status if {
+_status(task) = status if {
+	not task.status.conditions
 	status := task.status
+} else = status if {
+	some condition in task.status.conditions
+	condition.type == "Succeeded"
+	status := _slsav1_status(condition)
 } else = "MISSING"
+
+_slsav1_status(condition) := status if {
+	condition.status == "True"
+	status := "Succeeded"
+}
+
+_slsav1_status(condition) := status if {
+	condition.status == "False"
+	status := "Failed"
+}
+
+_pipelinerun := att if {
+	att := [a |
+		some a in lib.pipelinerun_attestations
+	]
+	count(att) > 0
```

Comment:
Why not just return an empty list?

Reply:
I get this error when trying that
```
eval_conflict_error: complete rules must not produce multiple outputs
```

Reply:
Yeah, https://github.com/enterprise-contract/ec-policies/pull/709#discussion_r1333468696 should address that.